python/cuopt/cuopt/linear_programming/problem.py

```diff
@@ -0,0 +1,384 @@
+import cuopt.linear_programming.data_model as data_model
+import cuopt.linear_programming.solver as solver
+import cuopt.linear_programming.solver_settings as solver_settings
+
+import enum
+import numpy as np
+
+# The type of a variable is either continuous, binary, or integer
+CONTINUOUS = 'C'
+BINARY = 'B'
+INTEGER = 'I'
+
+# Variable objects hold a reference to the problem they were created from as well as info
+# about there index, lower bound, upper bound, objective coefficient and variable type.
+# You can add and subtract Variables to scalars and other Variables to form LinearExpression objects.
+# You can multiply variables by a scalar to form a LinearExpression object
+class Variable:
```

Comment:
Lets add documentation underneath __init__ similar to other modules so these are fetched while building doc and also available while working interactively.

Reply:
added docs